frontend/src/App.tsx

```diff
@@ -32,12 +33,20 @@ import CodeEditor from "./pages/CodeEditor";
 import Contributors from "./pages/Contributors";
 import userBlock from "./hooks/userBlock";
 import Blocked from "./pages/Blocked";
+import PreLoader from "./components/Loader/Loader";
 // import axios from "axios";
 // axios.defaults.baseURL = "http://localhost:3001/";
 
 function App() {
   const { theme, toggleTheme } = useTheme();
   const { loading, isBlocked } = userBlock(5);
+  const [Preloading,setLoading]= useState(true);
```

Comment:
```suggestion
  const [preLoading,setPreLoading]= useState(true);
```

Variable is not in react naming convention
Similar to https://react.dev/reference/react/useState#setstate